### DISCUSSION TITLE
​Document lock function

### DISCUSSION THREAD

**Discussion by EnterMan123** [toeverything/AFFiNE#6727] [Ideas]
### Description

No editing can be done after locking

No more documents becoming cluttered, important information being deleted, etc. due to carelessness

This will become even more necessary if a mobile version is launched in the future

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by EnterMan123** [comment]
Although there is no mobile version currently available, I tried using AFFiNE on my mobile browser. A slight touch on the screen causes the content layout to change. I believe a locking feature is essential for the future.

**Comment by adm72** [comment]
Very useful feature ⚠️

**Comment by Beeditor04** [comment]
+1 feature, currently only support lock at Edgeless mode, need Lock function at Page mode as well
